core/src/main/java/org/apache/gravitino/utils/NamespaceUtil.java

```diff
@@ -107,6 +108,17 @@ public static Namespace ofFileset(String metalake, String catalog, String schema
     return Namespace.of(metalake, catalog, schema);
   }
 
+  /**
+   * Create a namespace for fileset from a schema name identifier.
+   *
+   * @param ident The schema name identifier.
+   * @return A namespace for fileset.
+   */
+  public static Namespace toFileset(NameIdentifier ident) {
```

Comment:
We could not get the actual meaning from the method name, what about `toFilesetNamespace` or something similar?
.
Parameter name `ident` can be replace with `schemaIdent` to improve the readiabitily.

Reply:
@Abyss-lord 
Have you resolved this comments? I noticed that you have marked it as resolved